### PR TITLE
Number of dropdown items and journey to game name-change

### PR DESCRIPTION
**Pull request by selmagudmundsen**
### PR Type 🍏

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] API
[x] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```

[Kanban Link 1](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=cccacba0c3d54befb1340165ed8f8355&pm=s)
[Kanban Link 2](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=2ae72e89a7ca4cfc842a0a3cb5aa7af2&pm=s)
[Kanban Link 3](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=a7be805ab67144b8a8e13b4168bc41b4&pm=s)

### Context 🤷‍♀️
After user testing the MVP we got come feedback for small UI changes that could be made to make the user experience of our admin panel more intuitive.

### What's new? 👶
- Changed all instances of "rute" to "spill".
- In the searchable dropdown for stop places  the display changed from 5 items to 8 in the fetch url.
- Added some more descriptive text under the _trekk vinner_ button to make it more clear.

### Screenshots 🖼️
Screenshot 2024-07-15 at 13 02 14
Screenshot 2024-07-15 at 13 06 40

### CODE REVIEW

File: package.json
Comment (maribsta) [inline package.json:1]: hvis denne ikke brukes kan den uninstalles

**Comment by tomaswedege** [comment]
Går det an å få sånn som den er i Entur-nettsiden? At det blir en scroll inne i dropdownen?

**Review by Magnus-Farstad — APPROVED** [review]
Kjempefantastisk arbeid!

**Review by maribsta — APPROVED** [review]
Veldig bra! Gjerne bare uninstalle packen i package og package lock

**Pull request merged**